code/modules/mob/living/carbon/human/human.dm

```diff
@@ -215,6 +215,13 @@
 	if(istype(MB))
 		MB.RunOver(src)
 
+	if(ishuman(AM) && lying)
+		var/mob/living/carbon/human/stampede = AM
+		visible_message("<span class='danger'>[src] is trampled by [stampede.name]!</span>", "<span class='userdanger'>You got trampled by [stampede.name]!</span>")
+		adjustBruteLoss(8)

```

Comment:
Yeah, was going to run armour and hit zones and stuff but didnt want to do that extra work if I couldn't get the rest of the mechanics to function